values-schema.yaml

```diff
@@ -1435,6 +1433,39 @@ definitions:
     required:
       - name
       - url
+  user:
+    type: object
+    description: A user can be a team member or an admin.
```

Comment:
Would it be good to mention that the user has to do with keycloak?

Reply:
Yes, that's right, that would be good. I added it in the description.